SEBrowser/Scripts/TSX/Components/EventSearch/EventSearchNavbar.tsx

```diff
@@ -73,7 +73,6 @@ const EventSearchNavbar = (props: IProps) => {
     
     const [showFilter, setFilter] = React.useState<('None' | 'Meter' | 'Asset' | 'AssetGroup' | 'Station')>('None');
     const [newEventCharacteristicFilter, setNewEventCharacteristicFilter] = React.useState<SEBrowser.IEventCharacteristicFilters>(null);
-    const [newTimeFilter, setNewTimeFilter] = React.useState<SEBrowser.IReportTimeFilter>(null);
```

Comment:
Removing this const as a go-between of the page components and timeFilter will break the page if you swap the tab to something else and then back, due to an infinte loop in state updates. This is because the slice loads the stored value out of session storage and tries to apply that to the components here, but the components attempt to change the filters due to intial conditions. Is there a reason this had to be removed?

Reply:
yes, adding this broke the ability to specify a date/time in the url. Because there is a race condition between the redux state being updated by parsing the query and the local state being updated by the previous redux state. That results in the new redux state (from the URL parsing) being overwritten by the local state. 
I don't think we need to load the Time filter from the local storage. 
Potential solution is to remove the local initial condition and set it to `null` and preventing the local state from overwriting the redux state with `null`

Reply:
I can't replicate this issue. Swapping Tabs works fine for me. @StephenCWills can you take a quick look and make sure this is resolved, since @gcsantos-gpa is out until next week?

Reply:
I also wasn't able to reproduce any infinite loop.